ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -156,6 +156,11 @@ object Config {
         .text("TLS: The crt file to be used as the trusted root CA.")
         .action((path, config) =>
           config.withTlsConfig(c => c.copy(trustCertCollectionFile = Some(new File(path)))))
+      opt[Boolean]("cert-revocation-checks")
+        .optional()
+        .text("TLS: Enable/disable certificate revocation checks with the OCSP. Default is to disable checks.")
+        .action((checksEnabled, config) =>
+          config.withTlsConfig(c => c.copy(revocationChecks = checksEnabled)))
```

Comment:
I think this too should be tested, i.e. it not only the cmdline arg for classic but also for KV.

Reply:
Do you mean a unit test for the argument? Or an integration test?

Reply:
I meant for the argument but now that you ask it looks to me like the integration test only covers sandbox-next, which is KV stack. Is there an IT for the classic (e.g. sandbox, DAML on SQL) stack?

Reply:
I haven't added an IT for the sandbox classic (didn't know if it's required still or not). Should I add ITs for the classic?

Reply:
@gerolf-da, can you comment on whether this is necessary?

My gut tells me that we need a unit test for the CLI argument everywhere it's used.

Off the top of my head, this needs to be:

  * DAML on SQL
  * kvutils/app
  * Sandbox
  * Sandbox Classic

I don't think we need more than one integration test.

Reply:
I think it should be tested at least for DAML-on-SQL which is based on the same stack as sandbox classic. Not really sure if the SDK sandbox nowadays is KV or classic but I think the latter. @gerolf-da or @SamirTalwar-DA WDYT?

Reply:
I'm fine if OCSP checking is only configurable for:
* DAML on SQL (`/ledger/daml-on-sql`)
* kvutils/app

The reason is that neither Sandbox nor Sandbox Classic will likely even be run by users with just TLS, let alone certificate revocation checks. DAML on SQL is a production ledger and the kvutils runner is also used by other ledger integrations.

Reply:
Another reason is to cut down on integration testing overhead.
@moritzkiefer-da: do you see a reason why certificate revocation should be part of the sandboxes?

Reply:
The only reason I could see is consistency. We have tls option in sandbox (arguably we shouldn’t have them but we probably don’t want to remove them) so it seems a bit weird to not make it configurable here.

But from a practical pov, I don’t expect anyone will run into this and if they do, telling them to use daml on sql seems perfectly fine.